ee/debug/checkups/checkups.go

```diff
@@ -120,6 +120,7 @@ func checkupsFor(k types.Knapsack, target targetBits) []checkupInt {
 		{&osqDataCollector{k: k}, doctorSupported | flareSupported},
 		{&osqRestartCheckup{k: k}, doctorSupported | flareSupported},
 		{&uninstallHistoryCheckup{k: k}, flareSupported},
+		{&desktopMenu{k: k}, doctorSupported | flareSupported},
```

Comment:
I'd skip doctor here. I can't really imagine it being meaningful

Reply:
done